pyiron_base/server/queuestatus.py

```diff
@@ -269,17 +270,23 @@ def update_from_remote(project, recursive=True):
             jobs_now_running_lst = []
         for job_id in df_combined.id.values:
             if job_id not in jobs_now_running_lst:
-                job = project.inspect(job_id)
-                state.queue_adapter.transfer_file_to_remote(
-                    file=job.project_hdf5.file_name,
-                    transfer_back=True,
-                    delete_remote=False,
-                )
-                status_hdf5 = job.project_hdf5["status"]
-                project.set_job_status(job_specifier=job.job_id, status=status_hdf5)
-                if status_hdf5 in job_status_finished_lst:
-                    job_object = job.to_object()
-                    job_object.transfer_from_remote()
+                try:
+                    job = project.inspect(job_id)
+                    state.queue_adapter.transfer_file_to_remote(
+                        file=job.project_hdf5.file_name,
+                        transfer_back=True,
+                        delete_remote=False,
+                    )
+                    status_hdf5 = job.project_hdf5["status"]
+                    project.set_job_status(job_specifier=job.job_id, status=status_hdf5)
+                    if status_hdf5 in job_status_finished_lst:
+                        job_object = job.to_object()
+                        job_object.transfer_from_remote()
+                except Exception as e:
```

Comment:
Can you be more explicit? What kind of error is raised here?

Reply:
The problem is that I had all kinds of errors already and don't remember all of them. But since there is tons of possible failure mechanisms I don't know if it really makes sense to explicitly list them.
Last error I encountered was a missing hdf5 file and my ugly workaround was this
jobs = pr.job_table()
jobs = jobs[(jobs.status=="running") | (jobs.status=="submitted")]
jobs
for ID in jobs.id:
    try:
        job = super_pr.load(ID)
        job.transfer_from_remote()
        job.status.collect = True
        job.run()
    except Exception as e:
        errors[ID] = e

Reply:
The issue with the except all errors part is that it makes it even harder for the next user to debug.

Reply:
Maybe as an option? 
if robust_fetch:
    catch all errors
else:
    current behavior

or I can add all errors I encounter one after another, but it will still be a list with like 5 different errors I guess.

Reply:
I would like to have the switch to change the behavior. Furthermore, I would suggest to use `state.logger.warning` instead of `warnings.warn` such that the warnings enter our pyiron log.
We could also think about returning a list of failed job_ids in the end (keeping None return value if everything was fine).

Reply:
I added an ignore_exceptions option and changed the warning